.travis.yml

```diff
@@ -42,4 +42,4 @@ deploy:
     condition: $TRAVIS_BRANCH =~ ^(master|dev)$
 
 after_success:
-  - ./gradlew jacocoRootReport coveralls
+  - CI_BRANCH=$TRAVIS_BRANCH ./gradlew jacocoRootReport coveralls
```

Comment:
Do we need to divide the two commands with &&, i.e., change the command to `CI_BRANCH=$TRAVIS_BRANCH && ./gradlew jacocoRootReport coveralls`

Reply:
I don't think it is necessary because "CI_BRANCH=$TRAVIS_BRANCH" is not a command but an environment variable declaration.